src/test_suite/test_runner_main.ts

```diff
@@ -162,155 +203,208 @@ export async function testRunnerMain(
-    console.log('-d <datapath>          Path to prix-fixe data files.');
-    console.log('                           attributes.yaml');
-    console.log('                           intents.yaml');
-    console.log('                           options.yaml');
-    console.log('                           products.yaml');
-    console.log('                           quantifiers.yaml');
-    console.log('                           rules.yaml');
-    console.log('                           stopwords.yaml');
-    console.log('                           units.yaml');
-    console.log(
-        '                       The -d flag overrides the value specified'
-    );
-    console.log(
-        '                       in the PRIX_FIXE_DATA environment variable.'
-    );
-    console.log('-h|help|?              Show this message.');
-    console.log('-s suite               Run tests in specified suite.');
-    console.log(
-        '-t <raw|stt|scoped>  Run tests using specified utterance field. RAW | STT | SCOPED'
-    );
-    console.log(
-        '                           The default is SCOPED and will use the highest corrected value provided in the yaml.'
-    );
-    console.log(
-        '                           RAW: force it to run on the original input, even if there are corrected versions available'
-    );
-    console.log(
-        '                             STT: if there is correctedSTT available use that otherwise use input, even if there is correctedScope available'
-    );
-    console.log(
-        '                          SCOPED: if there is correctedScope available use it, other wise fall back to correctedSTT and then input'
-    );
-    console.log(' ');
+            console.log('');
+            console.log('');
 
-    if (processorFactory.count() > 0) {
-        console.log('Available Processors:');
-        for (const processor of processorFactory.processors()) {
-            console.log(`  "-v=${processor.name}": ${processor.description}`);
-            // TODO: list expected data files for each processor.
+            suiteResults.results.forEach(r => allResults.recordResult(r));
         }
-    } else {
-        console.log('No Processors available.');
+
+        const testPassRate =
+            (allResults.passCount /
+                (allResults.passCount + allResults.failCount)) *
+            100;
+
+        console.log();
+        console.log('============================');
+        console.log(
+            `OVERALL RESULTS: ${allResults.passCount}/${allResults.results.length}`
+        );
         console.log(
-            'The supplied ProcessorFactory has no ProcessorDescriptions'
+            chalk`${allResults.results.length.toString()} tests run; {yellow.bold ${testPassRate.toString()}%} pass rate.`
         );
+        console.log('============================');
+        process.exit(0);
     }
 }
 
+const usage: Section[] = [
```

Comment:
So this being a global `const` means we pay a little price to create it each time even if nobody's asking for the usage. This affects start up performance which, in a CLI, you typically want to reduce as much as possible. 

I would suggest we actually create a little `showUsage` helper method which you call from above where you currently pass the `const` to `commandLineUsage` that just inlines this value into the call instead.